example/src/screens/Room.tsx

```diff
@@ -197,6 +199,7 @@ const styles = StyleSheet.create({
   },
   headerTitle: {
     marginLeft: 16,
+    maxWidth: '80%',
```

Comment:
Why 80%? Shouldn't it take just any available space? I can fit a few more letters easily here:
![Screenshot_20230510_145221](https://github.com/jellyfish-dev/react-native-membrane-webrtc/assets/6280457/7a1e4fbd-7bb3-42c4-ba10-dca4d8f25745)

Reply:
changed to flex